diagnostic.md

```diff
@@ -35,7 +35,9 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    The first route definition creates a different 'view state' that will generate new route 'product' that list one product from products. The product_id is nested under products, while in the second route definition this is not. The second route definition will list all the products on the page.
```

Comment:
These two route definitions are identical URL paths, and will both show a single product because they define `':product_id'`.  One of the big distinctions is where Ember will render the content.  
The nested `'products'/'product'` instructs Ember to render
`app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  
The non-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.